spec/lib/twingly/http_spec.rb

```diff
@@ -600,7 +632,7 @@ class CustomError < StandardError; end
 
   describe "#get", vcr: Fixture.example_org do
     include_examples "common HTTP behaviour for", :get, "example.org"
-
+    include_examples "verifies proxy functionality"
```

Comment:
To match how it looks in the other places in this file :)

```suggestion
    include_examples "verifies proxy functionality"

```